index.html

```diff
@@ -42,6 +44,162 @@ <h1 class="primary1">I'm Selma <br> Glad to see <br>you</h1>
           <li><a href=""><img src="images/Fing-vector.png" class="fing-vector" alt=""></a></li>
         </ul>
     </section>
-      
+
+    <!--Works Section-->
+    <section class="works">
+        <div class="card-works">
+            <img src="images/snapshot1.png"  alt="">
+            <div class="card-works-content1">
+            <h2 class="title tonic">Tonic</h2>
+            <ul class="frame">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <div class="desc-title">
+              <p class="primary-text2"> A daily selection of privately personalized reads;
+               no accounts or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javaScript</li>
+              </ul>
+            <button type="button" class="btn action">See Projects</button>
+            </div>
+            </div>
+        </div>
+    
+        <div class="card-works">
+          <img src="images/snapshot2.png"  alt="">
+          <div class="card-works-content2">
+            <h2 class="title multi-post">Multi-Post <br> Stories</h2>
+            <ul class="frame">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <div class="desc-title">
+              <p class="primary-text2"> A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+              <ul class="tags">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javaScript</li>
+              </ul>
+              <button type="button" class="btn action">See Projects</button>
+            </div>
+          </div>
+        </div>
+    
+        <div class="card-works">
+          <img src="images/snapshot3.png"  alt="">
+          <div class="card-works-content1">
+            <h2 class="title tonic">Tonic</h2>
+            <ul class="frame">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <div class="desc-title">
+              <p class="primary-text2"> A daily selection of privately personalized reads;
+                no accounts or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javaScript</li>
+              </ul>
+              <button type="button" class="btn action">See Projects</button>
```

Comment:
- [ ] kindly reduce your card's bottom margin and card gap a little, so it matches with the Figma version.
![Screenshot (312)_LI](https://user-images.githubusercontent.com/65298048/185624626-5e47323e-df42-4b81-a3b5-afc1f804531c.jpg)
- figma version
![Screenshot (314)](https://user-images.githubusercontent.com/65298048/185624968-1f34bfe9-bd31-48aa-ae42-d04800f541b6.png)